src/utils/wallet.ts

```diff
@@ -73,11 +73,11 @@ const wallet = {
       .toLowerCase();
     const wordsArray = newWordsString.split(' ');
 
-    const getInvalidWords = (words: string[]): string[] => {
+    const getInvalidWords = (_words: string[]): string[] => {
       const wordlist = Mnemonic.Words.ENGLISH;
       const errorList: string[] = [];
 
-      for (const word of words) {
+      for (const word of _words) {
```

Comment:
why?

Reply:
Because of the [`no-shadow`](https://eslint.org/docs/latest/rules/no-shadow) rule. This name was already declared in a scope above this one, so it should not be used again.

Reply:
As commented in another file `_words` may be interpreted as not being used, can we rename to `seed` or something else?